### PR TITLE
docs: update readme for MDX

### PR DESCRIPTION
**Pull request by rafegoldberg**
## 🧰 Changes

- [x] update the repo's readme for an MDX world
- [x] some minor housekeeping edits for consistency

### CODE REVIEW

**Review by kellyjosephprice — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -86,35 +76,33 @@ Extends [`CompileOptions`](https://mdxjs.com/packages/mdx/#compileoptions)
...
 
 ### `RunOpts`
 
 Extends [`RunOptions`](https://mdxjs.com/packages/mdx/#runoptions)
 
 ###### Additional Properties
 
-- `components` (`Record<string, MDXModule>`, optional) -- An object of tag names to executed components.
-- `imports` (`Record<string, unknown>`, optional) -- An object of modules to import.
+- `components` (`Record<string, MDXModule>`, optional) -- an object of tag names to executed components
```
Comment (mansagills) [inline README.md:90]: @rafegoldberg Is this supposed to be 'execute' or 'executed'?

File: README.md [hunk 2]
```diff
@@ -86,35 +76,33 @@ Extends [`CompileOptions`](https://mdxjs.com/packages/mdx/#compileoptions)
...
 ###### Additional Properties
 
-- `lazyImages` (`boolean`, optional) -- Load images lazily.
-- `safeMode` (`boolean`, optional) -- Extract script tags from `HTMLBlock`s
-- `components` (`Record<string, string>`, optional) -- An object of tag names to mdx.
-- `copyButtons` (`Boolean`, optional) — Automatically insert a button to copy a block of text to the clipboard. Currently used on `<code>` elements.
+- `lazyImages` (`boolean`, optional) -- load images lazily
+- `safeMode` (`boolean`, optional) -- extract script tags from `HTMLBlock`s
+- `components` (`Record<string, string>`, optional) -- an object of tag names to mdx.
+- `copyButtons` (`Boolean`, optional) — add a Copy button to code blocks
```
Comment (mansagills) [inline README.md:82]: Maybe lowercase 'copy' and italicize?

**Review by mansagills — COMMENTED** [review]
Just left a couple of comments, but looks good!

File: README.md (hunk 1 above)
Comment (rafegoldberg) [reply README.md]: T'was intentional, I think? But I updated the language to (hopefully) clarify!

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v7.4.2</b>](/readmeio/markdown/releases/tag/v7.4.2)
<br>

**Pull request merged**